pkgs/sketch_pad/lib/main.dart

```diff
@@ -35,6 +35,42 @@ import 'widgets.dart';
 
 const appName = 'DartPad';
 
+final router = GoRouter(
```

Comment:
Perhaps move the router creation into a method? This would then become something like `final GoRouter router = _createRouterConfig();`.